app/javascript/gobierto_dashboards/webapp/ViewerManager.vue

```diff
@@ -60,11 +61,22 @@ export default {
     } else {
       ({ data: this.dashboards } = await this.getDashboards({ context: this.context, data_pipe: this.pipe }))
     }
+
+    // Emit to his parent, if any
+    this.$emit(GobiertoEvents.DASHBOARD_LOADED, this.dashboards)
```

Comment:
Why if any? I don't see an if here, but you dispatching both events

Reply:
I meant _if there was any_, offering the chance to listen the event in a vue-way if this component was initialized by another vue component.